api/context.go

```diff
@@ -489,6 +510,46 @@ func Handle404(w http.ResponseWriter, r *http.Request) {
 	RenderWebError(err, w, r)
 }
 
+func GetSession(token string) *model.Session {
+	var session *model.Session
+	if ts, ok := sessionCache.Get(token); ok {
+		session = ts.(*model.Session)
+	}
+
+	if session == nil {
+		if sessionResult := <-Srv.Store.Session().Get(token); sessionResult.Err != nil {
+			l4g.Error("Invalid session token=" + token + ", err=" + sessionResult.Err.DetailedError)
+		} else {
+			session = sessionResult.Data.(*model.Session)
+		}
+	}
+
+	return session
+}
+
+func GetMultiSessionCookie(r *http.Request) []string {
```

Comment:
0/5 but maybe call this `GetMultiSessionCookieTokens` or something similar so it's obvious what's being returned?